resources/META-INF/plugin.xml

```diff
@@ -31,7 +31,7 @@
   </change-notes>
 
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-  <idea-version since-build="191.0"/>
+  <idea-version since-build="192.0"/>
```

Comment:
I think changing the supported version is what might be causing my issue. Do you need to change this for some reason?

Reply:
Are you testing with an older Intellij version?
I bumped the required version in order to be able to use the new properties introduced in intellij 2019.2: http://www.jetbrains.org/intellij/sdk/docs/reference_guide/ui_themes/themes_customize.html#customizing-the-color-of-ui-tabs

Reply:
I am actually testing with .3 so I don't think that's the issue. I think I'll have to release this as a major version update then, since it will break on older clients.